### DISCUSSION TITLE
Light theme design

### DISCUSSION THREAD

**Discussion by rustdesk** [rustdesk/rustdesk#3505] [General]
@NicKoehler @grummbeer 

![image](https://user-images.githubusercontent.com/71636191/222879762-5016ef63-aab2-4e01-98f5-b39c3c817641.png)

**Comment by rustdesk** [comment]
I made some changes
-  [x] thinner divider
-  [x] remove `Control remote desktop` background but with a border
-  [x] change original grayBg color from #EEEEEE to #EFEFF2

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/commit/122f0f961175dc46531a97714a124a17740ad816 https://github.com/rustdesk/rustdesk/commit/c11f1b3a511f556bd8de80056b0303cfe8ea14ea

**Comment by NicKoehler** [comment]
Looks good to me

**Comment by youhers** [comment]
Can the close button and minimize button of win be bigger
